libs/books/feature/src/lib/reading-list/reading-list.component.html

```diff
@@ -1,19 +1,35 @@
 <ng-container *ngIf="(readingList$ | async).length > 0; else empty">
   <div class="reading-list-item" *ngFor="let b of readingList$ | async">
+    <div>
+      <button *ngIf="!b?.finished" data-testing="mark-to-finish-reading-button" mat-icon-button color="magenta"
```

Comment:
For material buttons such as `mat-icon-button`, the `color` attribute takes a generic reference to a theme color e.g. `primary`, `accent`, `warn`